### ISSUE TITLE
Is files[] up to date?

### ISSUE THREAD

**Issue by andreygorbachevrabobank** [HowardHinnant/date#737]
Dear Sir/Madam,

I assume this variable lists data files from the tzdata downloadable. At the moment of writing that would be of the version 2022a. I think that the variable initialisation might have got our of sync with the tzdata2022a content, would you consider fixing it please?

Thank you,
Andrey

**Comment by HowardHinnant** [comment]
Just to make sure I'm understanding your comment, are you referring to the fact that the file systemv is no longer being distributed?

**Comment by andreygorbachevrabobank** [comment]
Yes, also pacificnew is not there any more I think. And I am not 100% sure if there are any new (additional) files which should be attempted to be loaded there now.

Thank you for getting back to me that quickly – your help is appreciated.

**Comment by HowardHinnant** [comment]
Ah, thanks, I missed pacificnew.  Yes, I'll remove these two from files[].

**Comment by andreygorbachevrabobank** [comment]
Cool, thank you. Do you think any new files should be taking their place? (I do not know enough about tzdata trajectory to understand this unfortunately).

**Comment by HowardHinnant** [comment]
I looked and didn't see any.  But then again I missed pacificnew. ;-)  If any file contains a non-commented-out keyword `Rule`, `Link`, `Leap` or `Zone`, we should consider including it.

**Comment by andreygorbachevrabobank** [comment]
I had a look at the files once more (with fresh eyes) and there might be 2 other files to be considered (not sure if they are new for 2022a - have not checked the older releases, just read through the 2022a files following your list of keywords):

1) "backzone" - but probably that is out of scope of what you want to do.

and

2) "factory" - which should assist with (consistent) default settings.
